.github/workflows/subclass.yaml

```diff
@@ -0,0 +1,40 @@
+name: Update Subclass Evidence
+
+on:
+#  schedule:
+#    - cron: '0 14 * * 1'
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container: obolibrary/odkfull:v1.5.1
```

Comment:
@matentzn need to update week of 22-Jul-2024 when ODK 1.5.2 is released.

Reply:
Already done #7985

Reply:
https://github.com/INCATools/ontology-development-kit/releases/tag/v1.5.2

Reply:
Do you not want to update the ODK version [here](https://github.com/monarch-initiative/mondo/pull/7801/files#diff-b785e7cd995fae5debc10cfbcfa1e93673fcaaf27871d9e2fe5c3dd662c2a58dR11)?

Reply:
good point thx for catching